<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDlg</class>
 <widget class="QDialog" name="SettingsDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QListWidget" name="contentsListWidget">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="iconSize">
      <size>
       <width>96</width>
       <height>96</height>
      </size>
     </property>
     <property name="movement">
      <enum>QListView::Static</enum>
     </property>
     <property name="spacing">
      <number>12</number>
     </property>
     <property name="viewMode">
      <enum>QListView::IconMode</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="behaviourPage">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="behaviourLabel">
           <property name="font">
            <font>
             <weight>75</weight>
             <italic>false</italic>
             <bold>true</bold>
             <underline>false</underline>
            </font>
           </property>
           <property name="text">
            <string>Behaviour preferences:</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="modeLabel">
             <property name="text">
              <string>Mode:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="modeComboBox"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0" colspan="2">
            <widget class="QLabel" name="setLabel">
             <property name="text">
              <string>Sets of words:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="originalLabel">
             <property name="text">
              <string>Language of the original:</string>
             </property>
             <property name="buddy">
              <cstring>originalLineEdit</cstring>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="languageOriginalComboBox">
             <property name="editable">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="translatedLabel">
             <property name="text">
              <string>Language of translation:</string>
             </property>
             <property name="buddy">
              <cstring>translatedLineEdit</cstring>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QComboBox" name="languageTranslatedComboBox">
             <property name="editable">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QListWidget" name="setsListWidget"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBoxTime">
           <property name="title">
            <string/>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="popupWindowIntervalLabel">
              <property name="text">
               <string>Pop-up window:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="staticTimeButton">
              <property name="text">
               <string>Static</string>
              </property>
              <attribute name="buttonGroup">
               <string>buttonGroup</string>
              </attribute>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="popupIntervalLabel">
                <property name="text">
                 <string>Popup interval (min):</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="popupIntervalSpinBox"/>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QRadioButton" name="randomTimeButton">
              <property name="text">
               <string>Random</string>
              </property>
              <attribute name="buttonGroup">
               <string>buttonGroup</string>
              </attribute>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLabel" name="popupIntervalFromLabel">
                  <property name="text">
                   <string>From (min):</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="popupIntervalFromSpinBox"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QLabel" name="popupIntervalToLabel">
                  <property name="text">
                   <string>To (min):</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="popupIntervalToSpinBox"/>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <widget class="Line" name="line_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="popupWindowIntervalLabel_2">
              <property name="text">
               <string>Tool tip:</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLabel" name="msgPopupIntervalLabel">
                <property name="text">
                 <string>Interval (min):</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="msgPopupIntervalDoubleSpinBox">
                <property name="decimals">
                 <number>3</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="msgDurationIntervalLabel">
                <property name="text">
                 <string>Duration (min):</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="msgDurationIntervalDoubleSpinBox">
                <property name="decimals">
                 <number>3</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>178</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="appearancePage">
        <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0,1">
         <item>
          <widget class="QLabel" name="appearanceLabel">
           <property name="font">
            <font>
             <weight>75</weight>
             <italic>false</italic>
             <bold>true</bold>
             <underline>false</underline>
            </font>
           </property>
           <property name="text">
            <string>Appearance preferences:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="tmpLabel">
           <property name="text">
            <string>TODO: appearance preferences</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="twFivePage">
        <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0,2">
         <item>
          <widget class="QLabel" name="twFiveLabel">
           <property name="font">
            <font>
             <weight>75</weight>
             <italic>false</italic>
             <bold>true</bold>
             <underline>false</underline>
            </font>
           </property>
           <property name="text">
            <string>25 Mode preferences:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="tmpLabel_2">
           <property name="text">
            <string>TODO: 25 mode preferences</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>435</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>321</x>
     <y>435</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
